app/src/pages/playground/playgroundUtils.ts

```diff
@@ -1168,16 +1168,23 @@ export function normalizeMessageContent(content?: unknown): string {
         // Stringify the result to ensure consistent formatting
         return JSON.stringify(secondParse, null, 2);
       }
-      // For regular strings, return as-is
-      return content;
-    } catch (e) {
-      // If parsing fails, return the original content
+    } catch {
+      // If parsing fails, fall through
+    }
+    // If the content is a valid non-string top level json value, return it as-is
+    // https://datatracker.ietf.org/doc/html/rfc7159#section-3
+    // 0-9 { [ null false true
+    // a regex that matches possible top level json values, besides strings
+    const nonStringStart = /^\s*[0-9{[\]false true null]/.test(content);
+    if (nonStringStart) {
       return content;
     }
   }
 
-  // For non-string content, stringify it with pretty printing
-  return JSON.stringify(content, null, 2);
+  // For any content that doesn't match the json spec for a top level value, stringify it with pretty printing
+  const out = JSON.stringify(content, null, 2);
+
+  return out;
```

Comment:
```suggestion
  return JSON.stringify(content, null, 2);
```

Reply:
ah yeah, I was logging this out. good catch